book/src/PythonBindings.md

```diff
@@ -0,0 +1,126 @@
+# Python Bindings
+
+Pravega Native client uses [PyO3](https://pyo3.rs) to generate Python bindings. 
```

Comment:
This should really go in a "building" section or similar as it's not really part of the API.

Reply:
ok, I have improved the doc.